assets/js/scripts.js

```diff
@@ -0,0 +1,241 @@
+        "opacity": {
+          "value": 0.5,
+          "random": false,
+          "anim": {
+            "enable": false,
+            "speed": 1,
+            "opacity_min": 0.1,
+            "sync": false
+          }
+        },
+        "size": {
+          "value": 10,
+          "random": true,
+          "anim": {
+            "enable": false,
+            "speed": 40,
+            "size_min": 0.1,
+            "sync": false
+          }
+        },
+        "line_linked": {
+          "enable": true,
+          "distance": 150,
+          "color": "#bfbbae",
+          "opacity": 0.4,
+          "width": 1
+        },
+        "move": {
+          "enable": true,
+          "speed": 6,
+          "direction": "none",
+          "random": false,
+          "straight": false,
+          "out_mode": "out",
+          "bounce": false,
+          "attract": {
+            "enable": false,
+            "rotateX": 600,
+            "rotateY": 1200
+          }
+        }
+      },
+      "interactivity": {
+        "detect_on": "canvas",
+        "events": {
+          "onhover": {
+            "enable": true,
+            "mode": "grab"
+          },
+          "onclick": {
+            "enable": true,
+            "mode": "push"
+          },
+          "resize": true
+        },
+        "modes": {
+          "grab": {
+            "distance": 140,
+            "line_linked": {
+              "opacity": 1
+            }
+          },
+          "bubble": {
+            "distance": 400,
+            "size": 40,
+            "duration": 2,
+            "opacity": 8,
+            "speed": 3
+          },
+          "repulse": {
+            "distance": 200,
+            "duration": 0.4
+          },
+          "push": {
+            "particles_nb": 4
+          },
+          "remove": {
+            "particles_nb": 2
+          }
+        }
+      },
+      "retina_detect": true
+    });
+  }
+
+
+  function enablei18n(setLang) {
+   'use strict';
+   var i18n = $.i18n();
+   var lang = 'en';
+
+   if(setLang == false){
+     var sessionLang = sessionStorage.getItem('dgc-lang');
+     if(typeof sessionLang == "undefined" || sessionLang == null || sessionLang == "null" || sessionLang == ""){
+       var userLang = navigator.language || navigator.userLanguage; 
+       if(typeof userLang == "undefined" || userLang == ""){
+        userLang = 'en-EN';
+      }
+      if(userLang.indexOf('-')){
+        lang = userLang.split('-')[0];
+      }
+    }
+    else{
+      lang = sessionLang;
+    }
+  }
+  else{
+    lang = setLang;
+  }
+
+  if(lang != 'en' && lang != 'de' && lang != 'fr' && lang != 'es'){
+    lang = 'en';
+  }
+  sessionStorage.setItem('dgc-lang', lang);
+
+  i18n.locale = lang;
+  
+  i18n.load('assets/i18n', i18n.locale).done(
+    function() {
+     $('[data-i18n]').each(function(index) {
+      $(this).i18n();
+      $(this).html($.i18n($(this).attr('data-i18n')));
+    } );
+   } );
+}
+// Enable debug
+$.i18n.debug = true;
```

Comment:
I think we should disable the debug mode?

Reply:
good catch... I missed that one, thanks! I'll PR that separately.

Reply:
> good catch... I missed that one, thanks! I'll PR that separately.

sorry missed your comment and updated already